src/document.js

```diff
@@ -500,6 +523,57 @@ const documentsMain = {
 				$(document.body).removeClass('claro')
 			})
 		},
+
+		initiateOpenLocally() {
+			OC.dialogs.confirmDestructive(
```

Comment:
Could we maybe only issue the confirm action if there are other users present in the document? The list of views should be available.

Reply:
I had it that way originally but changed it because the user might accidentally click it (when trying to save for example), and they might panic when the document suddenly closes.

Also we might want to add in the dialog that it only works if the desktop client (v3.6.0+) is installed and running